index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
If you're sure they'll all be binary - what about unary (only think of awful things like, stop at depth 3, or compile with feature flog or altered config for this node) or variadic (`+` works well here)